catwatch-backend/etc/scm/scm-source.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+REV=$(git rev-parse HEAD)
+URL=$(git config --get remote.origin.url)
```

Comment:
This works for both the repo cloned directly and cloned from a GitHub fork, right?